src/main/java/today/sesac/shoutify/post/dto/request/PostCreateRequest.java

```diff
@@ -0,0 +1,15 @@
+package today.sesac.shoutify.post.dto.request;
+
+import lombok.Getter;
+
+@Getter
+public class PostCreateRequest {
```

Comment:
P2
마지막에 Dto까지 붙여주세욥~
[여기](https://www.notion.so/hannanana/package-package-21033717f9138006898acbaf97a109e2?source=copy_link) ranking/dto/response 참고해 주세욥
```suggestion
public class PostCreateRequestDto {
```